src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -244,6 +244,9 @@ frmelmnts.lbl.tcno=No
 frmelmnts.lbl.experience=Experience
 frmelmnts.lbl.dashboardnoorgselected=No organization selected!
 frmelmnts.lbl.dashboardnoorgselecteddesc=Please select a organization from the above list
+frmelmnts.lbl.more= more
+frmelmnts.lbl.showLess=Show less
+frmelmnts.lbl.profilePopup= Enter details to get relevant content
```

Comment:
What is the significance of blank space after = in frmelmnts.lbl.more and frmelmnts.lbl.profilePopup?